doc/Command-Reference.md

```diff
@@ -4088,6 +4092,170 @@ This command deletes the SNMP Trap server IP address to which SNMP agent is expe
+3) optional - --json option to display the result in json format. By default output will be in tabular format.
+
+- Example:
+  ```
+  admin@sonic:~$ show muxcable config
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config --json
+{
+    "MUX_CABLE": {
+        "PEER_TOR": "10.1.1.1",
+        "PORTS": {
+            "Ethernet0": {
+                "STATE": "active",
+                "SERVER": {
+                    "IPv4": "10.1.1.1",
+                    "IPv6": "fc00::75"
+                }
+            }
+        }
+    }
+}
+
+  admin@sonic:~$ show muxcable config Ethernet0
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config Ethernet0 --json
+{
+    "MUX_CABLE": {
+        "PORTS": {
+            "Ethernet0": {
+                "STATE": "active",
+                "SERVER": {
+                    "IPv4": "10.1.1.1",
+                    "IPv6": "fc00::75"
+                }
+            }
+        }
+    }
+}
+  ```
+
+
+### Muxcable Config commands
+
+
+**config muxcable mode**
+
+This command is used for setting the configuration of a muxcable Port/all ports to be active or auto. The user has to enter a port number or else all to make the muxcable config operation on all the ports. Depending on the status of the muxcable port state the resultant output could be OK or INPROGRESS or FAILED. OK would imply no change on the state, INPROGRESS would mean the toggle is happening in the background. FAILED would mean that the muxcable is in bad state or the operation is not successfull.
+
+- Usage:
+  ```
+  config muxcable mode [OPTIONS] <operation_status> <port_name>
+  ```
+
+While configuring the muxcable, users need to configure the following fields for the operation
+1) operation_status - Permitted operation to be configured can only be auto or active
+2) port_name        - Port name could be a valid port or all to which the muxcable operation would occur. All would imply the operation on all the ports
+3) optional         - --json option to display the result in json format. By default output will be in tabular format.
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config muxcable  mode active Ethernet0
+  port       state
+  ---------  -------
+  Ethernet0  OK
+
+  admin@sonic:~$ sudo config muxcable  mode --json active Ethernet0
+{
+    "Ethernet0": "OK"
+}
+
+  admin@sonic:~$ sudo config muxcable  mode active all
+  port        state
+  ----------  ----------
+  Ethernet0   OK
+  Ethernet32  INPROGRESS
+
+  admin@sonic:~$ sudo config muxcable  mode active all --json
+{
+    "Ethernet32": "INPROGRESS",
+    "Ethernet0": "OK"
+}
```

Comment:
Indent 2 spaces to align with block to fix formatting.

Reply:
fixed